styles.css

```diff
@@ -267,6 +269,67 @@ header {
   height: 6vw;
 }
 
+.popup-mobile {
+  display: none;
+  width: 81.3vw;
+  height: auto;
+  margin-top: 10vw;
+}
+
+.popup-desktop {
+  display: none;
+}
+
+.description {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+}
+
+#closexdescription {
+  margin-left: 83vw;
+}
+
```

Comment:
- I suggest positioning your popup as fixed to make sure the user can't lose it when he scrolls vertically :recycle: 
[Screencast from 20-04-2023 23:14:55.webm](https://user-images.githubusercontent.com/97834160/233489989-80f629c7-cdc4-4c7c-aa10-5ab23cc168ae.webm)